form.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="utf-8">
+		<title>Mental Rental - Add Information</title>
+		<link rel="stylesheet" href="style.css" charset="utf-8">
+		<link href="https://fonts.googleapis.com/css?family=Mitr" rel="stylesheet">
+		<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.1.6/Chart.min.js">
+
+		</script>
+	</head>
+	<body>
+		<header>
+			<section id="connect">
+				<p>Connect:</p>
+				<a id="githubLink"href="https://github.com/olds1423/finalProject"><img id="github"  src="assets/style/github.png"/></a>
+			</section>
+
+			<!-- Title -->
+			<h1>Add/Update Information</h1>
+
+			<!-- login -->
+			<a id="login">Log In/ Register</a>
+		</header>
+
+		<nav id="navbar">
+
+			<a id="formLink" href="index.html">Home</a>
+			<a id="statsLink" href="statistics.html">All Statistics</a>
+
+		</nav>
+
+
+		<main id="main">
+			<form id='updateForm' method="post">
```

Comment:
The `method='post'` probably auto-completed? It's unnecessary in this case since you're not actually posting your form to a data storage source.